src/layouts/partial-cards.html

```diff
@@ -64,7 +64,7 @@
               on-tap='scrollToTop'
             >
               <template is='dom-if' if='[[!defaultView]]'>
-                [[locationName]]
+                default_view
```

Comment:
What? no. This makes no sense.

Before continuing this PR, why don't you make a list of all the different permutations that the header can be in and include screenshots.

Reply:
here we go: https://i.imgur.com/tFTJqFX.png